src/routes/docs/components/skeleton.md

```diff
@@ -183,7 +183,7 @@ The component has the following props, type, and default values. See <A href="/d
 ## References
 
 <P>
-  <A href="https://flowbite.com/docs/components/skeleton/" target="_blank" rel="noreferrer" class="link"
+  <A href="https://flowbite.com/docs/components/skeleton/" class="link"
```

Comment:
All those References should be changed to markdown as:
```[Tailwind CSS Skeleton](https://flowbite.com/docs/components/skeleton)```

Reply:
@jjagielka this is how they were before, but sure, I can update them to use markdown

Reply:
It's not urgent. Just an observation. Using markdown will unify the look and feel in the long term though.

Reply:
@jjagielka I've converted all of them to markdown + fixed some wrong text/links too